builder/docker/config.go

```diff
@@ -54,6 +55,8 @@ type Config struct {
 func NewConfig(raws ...interface{}) (*Config, []string, error) {
 	c := new(Config)
 
+	c.FixUploadOwner = true
```

Comment:
I like this method for defaulting a bool to true a lot better than what I came up with previously 😉 https://github.com/hashicorp/packer/blob/e497c6027fbd1e0dbc0914f8404e6c7dca7c3f72/provisioner/shell/provisioner.go#L107-L110